README.md

```diff
@@ -1,8 +1,6 @@
-# The Software Package Data Exchange (SPDX®) Specification
+# The System Package Data Exchange (SPDX®) Specification
 
-[![Build Status](https://travis-ci.org/spdx/spdx-spec.svg?branch=master)](https://travis-ci.org/spdx/spdx-spec)
-
-The Software Package Data Exchange® (SPDX®) specification is a standard format for communicating the components, licenses and copyrights associated with software packages.
+The System Package Data Exchange (SPDX®) specification is an open standard capable of representing systems with software components in as SBOMs (Software Bill of Materials) and other AI, data and security references supporting a range of risk management use cases.
```

Comment:
Suggest we simplify this sentence unless this is previously agreed up on verbiage?

Reply:
Lines up with the discussion of what went on the banner of the spdx.dev site that happended a couple of weeks ago.